<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:	#E0E0E0;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Img">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
border-color:black;
border-width:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="resultImg">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
border-color:black;
border-width:2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>491</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>確認</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>491</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_1">
         <property name="styleSheet">
          <string notr="true">font-size:20px;
color:red;</string>
         </property>
         <property name="text">
          <string>圖片檔案位置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="P_path">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browse">
         <property name="styleSheet">
          <string notr="true">background-color:#ADADAD;
width:50px;
 border-radius:8px;
font-family:Microsoft Yahei;
</string>
         </property>
         <property name="text">
          <string>瀏覽</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="rotate">
         <property name="text">
          <string>旋轉</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>左旋</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="low"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>度~</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="high"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>度</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="bright">
         <property name="text">
          <string>亮度:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="min_brightness"/>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>~</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="max_brightness"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>對比度:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="min_contrast"/>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>~</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="max_contrast"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QCheckBox" name="resize">
         <property name="text">
          <string>缩放</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>寬度:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="min_width"/>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>~</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="max_width"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>高度:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="min_height"/>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>~</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="max_height"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QCheckBox" name="shift">
         <property name="text">
          <string>平移</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>右移:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="min_shift_right"/>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>~</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="max_shift_right"/>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>下移:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="min_shift_down"/>
       </item>
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>~</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="max_shift_down"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QCheckBox" name="flip">
         <property name="text">
          <string>鏡射</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="min_flipValue"/>
       </item>
       <item>
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>~</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="max_flipValue"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>1: 水平鏡射 0: 垂直鏡射 -1: 水平、垂直同時</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QCheckBox" name="hue">
         <property name="text">
          <string>調色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>照片張數:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pic_count"/>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>色調區間</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hue_low"/>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>~</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hue_high"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>107</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>結果</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseResult">
       <property name="text">
        <string>瀏覽</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>107</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>原圖</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseImg">
       <property name="text">
        <string>瀏覽</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
